index.html

```diff
@@ -0,0 +1,243 @@
+
+
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="/Users/roobadi/Desktop/github/JS01Anaiya/STYLES/styles.css">
+</head>
+<body>
+  <h1>Harry Potter is cool!</h1>
+  <p>
+Facebook
+Join or Log Into Facebook  
+Email or Phone
+
+Password
+
+Forgot account?
+Log In
+Do you want to join Facebook?
+Sign Up
+Sign Up
+100 reasons why Harry Potter is awesome
+April 20, 2011 at 11:03pm
+1.	JK Rowling might not be the BEST writer in the world, but she is still very, very good.
```

Comment:
Also suggest that you used an **ordered list** to make your numbered list. Your code would be something like this:

``` html
<ol>
   <li>The first thing on my list</li>
   <li>The second thing on my list</li>
   <li>Notice that I'm not including the numbers</li>
   <li>They are included automatically because this is an ordered list!</li>
</ol>
```